<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DrumPadPresetDetailWidget</class>
 <widget class="QScrollArea" name="DrumPadPresetDetailWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>389</width>
    <height>774</height>
   </rect>
  </property>
  <property name="widgetResizable">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="scrollAreaWidgetContents">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>387</width>
     <height>772</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="labelId">
      <property name="text">
       <string>id:</string>
      </property>
      <property name="buddy">
       <cstring>lineEditId</cstring>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="labelName">
      <property name="text">
       <string>name:</string>
      </property>
      <property name="buddy">
       <cstring>lineEditName</cstring>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="labelAuthor">
      <property name="text">
       <string>author:</string>
      </property>
      <property name="buddy">
       <cstring>lineEditAuthor</cstring>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="labelOrderBy">
      <property name="text">
       <string>orderBy:</string>
      </property>
      <property name="buddy">
       <cstring>lineEditOrderBy</cstring>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="labelVersion">
      <property name="text">
       <string>version:</string>
      </property>
      <property name="buddy">
       <cstring>lineEditVersion</cstring>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="labelTempo">
      <property name="text">
       <string>tempo:</string>
      </property>
      <property name="buddy">
       <cstring>spinBoxTempo</cstring>
      </property>
     </widget>
    </item>
    <item row="6" column="0">
     <widget class="QLabel" name="labelIcon">
      <property name="text">
       <string>icon:</string>
      </property>
      <property name="buddy">
       <cstring>lineEditIcon</cstring>
      </property>
     </widget>
    </item>
    <item row="7" column="0">
     <widget class="QLabel" name="labelPrice">
      <property name="text">
       <string>price:</string>
      </property>
      <property name="buddy">
       <cstring>spinBoxPrice</cstring>
      </property>
     </widget>
    </item>
    <item row="8" column="0">
     <widget class="QLabel" name="labelPriceForSession">
      <property name="text">
       <string>priceForSession:</string>
      </property>
      <property name="buddy">
       <cstring>spinBoxPriceForSession</cstring>
      </property>
     </widget>
    </item>
    <item row="9" column="0">
     <widget class="QLabel" name="labelHasInfo">
      <property name="text">
       <string>hasInfo:</string>
      </property>
      <property name="buddy">
       <cstring>checkBoxHasInfo</cstring>
      </property>
     </widget>
    </item>
    <item row="10" column="0">
     <widget class="QLabel" name="labelTags">
      <property name="text">
       <string>tags:</string>
      </property>
      <property name="buddy">
       <cstring>listViewTags</cstring>
      </property>
     </widget>
    </item>
    <item row="11" column="0">
     <widget class="QLabel" name="labelDeleted">
      <property name="text">
       <string>DELETED:</string>
      </property>
      <property name="buddy">
       <cstring>checkBoxDeleted</cstring>
      </property>
     </widget>
    </item>
    <item row="12" column="0">
     <widget class="QLabel" name="labelDifficulty">
      <property name="text">
       <string>difficulty:</string>
      </property>
      <property name="buddy">
       <cstring>spinBoxDifficulty</cstring>
      </property>
     </widget>
    </item>
    <item row="13" column="0">
     <widget class="QLabel" name="labelSample">
      <property name="text">
       <string>sample:</string>
      </property>
      <property name="buddy">
       <cstring>spinBoxSample</cstring>
      </property>
     </widget>
    </item>
    <item row="14" column="0">
     <widget class="QLabel" name="labelAudioPreview1Name">
      <property name="text">
       <string>audioPreview1Name:</string>
      </property>
      <property name="buddy">
       <cstring>lineEditAudioPreview1Name</cstring>
      </property>
     </widget>
    </item>
    <item row="15" column="0">
     <widget class="QLabel" name="labelAudioPreview1URL">
      <property name="text">
       <string>audioPreview1URL:</string>
      </property>
      <property name="buddy">
       <cstring>lineEditAudioPreview1URL</cstring>
      </property>
     </widget>
    </item>
    <item row="16" column="0">
     <widget class="QLabel" name="labelAudioPreview2Name">
      <property name="text">
       <string>audioPreview2Name:</string>
      </property>
      <property name="buddy">
       <cstring>lineEditAudioPreview2Name</cstring>
      </property>
     </widget>
    </item>
    <item row="17" column="0">
     <widget class="QLabel" name="labelAudioPreview2URL">
      <property name="text">
       <string>audioPreview2URL:</string>
      </property>
      <property name="buddy">
       <cstring>lineEditAudioPreview2URL</cstring>
      </property>
     </widget>
    </item>
    <item row="18" column="0">
     <widget class="QLabel" name="labelImagePreview1">
      <property name="text">
       <string>imagePreview1:</string>
      </property>
      <property name="buddy">
       <cstring>lineEditImagePreview1</cstring>
      </property>
     </widget>
    </item>
    <item row="19" column="0">
     <widget class="QLabel" name="labelVideoPreview">
      <property name="text">
       <string>videoPreview:</string>
      </property>
      <property name="buddy">
       <cstring>lineEditVideoPreview</cstring>
      </property>
     </widget>
    </item>
    <item row="20" column="0">
     <widget class="QLabel" name="labelVideoTutorial">
      <property name="text">
       <string>videoTutorial:</string>
      </property>
      <property name="buddy">
       <cstring>lineEditVideoTutorial</cstring>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="lineEditId">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="lineEditName">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="lineEditAuthor">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QLineEdit" name="lineEditOrderBy">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QLineEdit" name="lineEditVersion">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QSpinBox" name="spinBoxTempo">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="minimum">
       <number>-2147483647</number>
      </property>
      <property name="maximum">
       <number>2147483647</number>
      </property>
     </widget>
    </item>
    <item row="6" column="1">
     <widget class="QLineEdit" name="lineEditIcon">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="7" column="1">
     <widget class="QSpinBox" name="spinBoxPrice">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="minimum">
       <number>-2147483647</number>
      </property>
      <property name="maximum">
       <number>2147483647</number>
      </property>
     </widget>
    </item>
    <item row="8" column="1">
     <widget class="QSpinBox" name="spinBoxPriceForSession">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="minimum">
       <number>-2147483647</number>
      </property>
      <property name="maximum">
       <number>2147483647</number>
      </property>
     </widget>
    </item>
    <item row="9" column="1">
     <widget class="QCheckBox" name="checkBoxHasInfo">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>CheckBox</string>
      </property>
     </widget>
    </item>
    <item row="11" column="1">
     <widget class="QCheckBox" name="checkBoxDeleted">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>CheckBox</string>
      </property>
     </widget>
    </item>
    <item row="12" column="1">
     <widget class="QSpinBox" name="spinBoxDifficulty">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="minimum">
       <number>-2147483647</number>
      </property>
      <property name="maximum">
       <number>2147483647</number>
      </property>
     </widget>
    </item>
    <item row="13" column="1">
     <widget class="QSpinBox" name="spinBoxSample">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="minimum">
       <number>-2147483647</number>
      </property>
      <property name="maximum">
       <number>2147483647</number>
      </property>
     </widget>
    </item>
    <item row="14" column="1">
     <widget class="QLineEdit" name="lineEditAudioPreview1Name">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="15" column="1">
     <widget class="QLineEdit" name="lineEditAudioPreview1URL">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="16" column="1">
     <widget class="QLineEdit" name="lineEditAudioPreview2Name">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="17" column="1">
     <widget class="QLineEdit" name="lineEditAudioPreview2URL">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="18" column="1">
     <widget class="QLineEdit" name="lineEditImagePreview1">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="19" column="1">
     <widget class="QLineEdit" name="lineEditVideoPreview">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="20" column="1">
     <widget class="QLineEdit" name="lineEditVideoTutorial">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="22" column="0">
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="21" column="0">
     <widget class="QLabel" name="labelTimestamp">
      <property name="text">
       <string>Timestamp:</string>
      </property>
      <property name="buddy">
       <cstring>dateTimeEditTimestamp</cstring>
      </property>
     </widget>
    </item>
    <item row="21" column="1">
     <widget class="QDateTimeEdit" name="dateTimeEditTimestamp">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="10" column="1">
     <widget class="QListView" name="listViewTags">
      <property name="enabled">
       <bool>false</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
